azure_jumpstart_ag/artifacts/settings/ESA/pv.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+    ### CREATE A NAME HERE ###
+    name: esaPVName
+    ### USE A NAMESPACE THAT MATCHES YOUR INTENDED CONSUMING POD OR "DEFAULT" ###
+    namespace: namespace
+spec:
+    capacity:
+        ### This storage capacity value is not enforced at this layer. ###
+        storage: 10Gi
```

Comment:
Should we make this configurable by user?

Reply:
It's parametrizable via automation now with placeholders